cg-spec/editors_draft.html

```diff
@@ -2301,8 +2302,9 @@ <h3>sparql-compare</h3>
           error, based on the comparison of two
           <a href="#dfn-rdf-star-terms">RDF-star terms</a>.
           When comparing two RDF-star triple terms, comparison is 
-          by pairwise comparison of subject, then predicate, then object.
-        </p>
+          by pairwise comparison of components subject, then predicate, then object.
```

Comment:
```suggestion
          by pairwise comparison of their components, first their subject, next their predicate, finally their object.
```

Reply:
I hope you find this better:
```
          When comparing two RDF-star
          triple terms, comparison is by pairwise comparison of the components: first
          compare the subjects, then compare the predicates, then compare the objects.
```

Reply:
This text is supposed to just giving an outline if the process, with the definition in the box.

I'll reword it - maybe just describing it as "pairwise" is enough.